config/locales/alchemy.it.yml

```diff
@@ -0,0 +1,924 @@
+      '1024': '1024px (iPad - Landscape)'
+      '1280': '1280px (Desktop)'
+    recently_uploaded_only: 'Solo caricate recentemente'
+    "regular method": "Metodo normale"
+    remove: "Rimuovi"
+    rename_file: "Rinomina questo file."
+    rename: "Rinomina"
+    replace: sostituisci
+    'Replaced Tag': "Il tag '%{old_tag}' è stato sostiuito con '%{new_tag}'"
+    resources:
+      relation_select:
+        blank: '- vuoto -'
+    right: "destra"
+    robot_follow: "i robot possono seguire i link."
+    robot_index: "permetti ai robot di indicizzare."
+    save: "Salva"
+    "save order": "Salva ordine"
+    saved_link: "Link salvato."
+    search: "cerca"
+    search_engines: "Motori di ricerca"
+    select_element: "Seleziona elemento"
+    seperate_tags_with_comma: "Separa i tag con una virgola"
+    show_element_content: "Mostra contenuto di questo elemento."
+    show_elements_from_page: "Mostra tutti gli elementi di questa pagina"
+    show_eq: "Mostra EQ"
+    show_navigation: "Mostra in navigazione"
+    show_page_in_sitemap: "Mostra la pagina nella sitemap."
+    signup_mail_delivery_error: 'Non è stato possibile consegnare la mail di signup. Ricontrolla le impostazioni dell''email.'
+    small_thumbnails: "Thumbnail piccoli"
+    subject: "Soggetto"
+    successfully_added_element: "Aggiunto nuovo elemento con successo."
+    successfully_deleted_tag: "Tag eliminato con successo"
+    successfully_restored_element: "Elemento ripristinato con successo."
+    successfully_saved_element_position: "Posizione dell'elemento aggiornata con successo."
+    successfully_updated_tag: "Tag aggiornato con successo"
+    swap_image: "Modifica immagine"
+    insert_image: "Inserisci immagine"
+    tag_list: Tag
+    tags_get_created_if_used_the_first_time: 'I tag vengono creati se usati per la prima volta.'
+    this_picture_is_used_on_these_pages: "L'immagine è usata nelle seguenti pagine"
+    title: "Titolo"
+    to_alchemy: "Torna ad Alchemy"
+    "trash element": "Metti l'elemento nel cestino"
+    unknown: "sconosciuto"
+    unlink: "Rimuovi link da questa immagine."
+    unlock_page: "Lascia la pagina"
+    unlocked_page: "Pagina %{name} sbloccata."
+    upload_file: "Upload (uno/più) file"
+    upload_image: "Upload (uno/più) immagini"
+    url_name: "URL-Name"
+    visible: "visibile"
+    want_to_create_new_language: "Vuoi creare un nuovo albero della lingua vuoto?"
+    want_to_make_copy_of_existing_language: "Vuoi copiare un albero della lingua esistente?"
+    "We need at least one default.": "Deve esistere almeno una lingua di default."
+    welcome_note: "Benvenuto %{name}"
+    welcome_back_note: "Bentornato %{name}"
+    welcome_please_identify_notice: "Benvenuto! Prego identificati."
+    width: "Larghezza"
+    without_tag: "Senza tag"
+    you_can_rename_this_tag: "Puoi rinominare questo tag"
+    zoom_image: "Effettua lo zoom di questa immagine"
+    "Leave Alchemy": "Abbandona Alchemy"
```

Comment:
If this line change to `Esci da` I think we have also to change the next two lines with `esci` and `Stai per uscire da Alchemy`